.bazelrc

```diff
@@ -20,4 +20,65 @@ test --flaky_test_attempts=3
 
 # The unified launcher runs in Python 2 host configuration
 # https://github.com/bazelbuild/bazel/issues/7899
-build --host_force_python=PY2 
\ No newline at end of file
+build --host_force_python=PY2 
+
+# ------------------------
+# General RBE configuration
+# ------------------------
+build:remote --jobs=50
+
+# Java toolchain setup
+build:remote --java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_hostjdk8
+build:remote --host_javabase=@rbe_default//java:jdk
+build:remote --javabase=@rbe_default//java:jdk
+
+# C++ toolchain setup
+build:remote --extra_toolchains=@rbe_default//config:cc-toolchain
+build:remote --crosstool_top=@bazel_toolchains//configs/ubuntu16_04_clang/9.0.0/bazel_1.0.0/cc:toolchain
+
+# Remote instance setup
+build:remote --remote_instance_name=projects/bazel-untrusted/instances/default_instance
+build:remote --project_id=bazel-untrusted
+
+# Set various strategies so that all actions execute remotely. Mixing remote
+# and local execution will lead to errors unless the toolchain and remote
+# machine exactly match the host machine.
+build:remote --spawn_strategy=remote
+build:remote --strategy=Javac=remote
+build:remote --strategy=Closure=remote
+build:remote --strategy=Genrule=remote
```

Comment:
This should no longer be necessary, because Bazel no longer overrides the spawn strategy internally for these three mnemonics.